manager/Makefile

```diff
@@ -27,13 +27,8 @@ source-build:
 # Overwrite docker-build from docker.mk
 .PHONY: docker-build
 docker-build: source-build
-ifdef FROM_IMAGE
-	docker build . -t ${IMG} -f Dockerfile --build-arg image=${FROM_IMAGE}
-	rm manager
-else
 	docker build . -t ${IMG} -f Dockerfile
```

Comment:
docker build . -t ${IMG} -f Dockerfile --build-arg tag=${BASE_IMAGE_TAG}

and to add `BASE_IMAGE_TAG=8.7` into `requirements.env`